mondey_backend/src/mondey_backend/models/questions.py

```diff
@@ -8,85 +8,112 @@
 from .utils import fixed_length_string_field
 
 
-class UserQuestionTextBase(SQLModel):
+# user questions
+class QuestionTextBase(SQLModel):
     question: str = ""
     options_json: str = ""
+    options: str = ""
+
+
+class Question(SQLModel):
+    order: int = 0
+    component: str = "select"
+    type: str = "text"
+    options: str = ""
+    additional_option: str = ""
```

Comment:
that would be cool, but there is a hurdle: Unless I misremember, the `QuestionText` element only localizes the names of the values to be selected, not the values themselves. In the frontend, what is bound to are the selected values, however. So first,  for the admin interface, the change wouldn´t make much of a  difference I think because we have to localize the names of the options there, anyway, and what is presented to the user is localized accordingly - they don´t see the actual values.

To make the suggestion work, we would need to add logic to translate the selected value to the localized element (even if it's just an `indexOf` call). Alternatively the logic of how the textfield for non-standard options is made to appear could be changed somehow to not rely on the selected value but I don´t really see how unless the textfield for additional stuff should always be visible.